src/maintenance/azext_maintenance/azext_metadata.json

```diff
@@ -1,4 +1,4 @@
 {
-    "azext.isPreview": true,
-    "azext.minCliCoreVersion": "2.3.1"
+    "azext.isExperimental": true,
```

Comment:
You cannot change it from isPreview to isExperimental.

Reply:
Remove isExperimenatal and isPreview. This is GA release for maintenance sevice.